Description: Fix libreoffice profile
 firejail (0.9.58.2-3parrot4) testing; urgency=medium
 .
   * Allow interpreters in libreoffice profile.
Author: Lorenzo "Palinuro" Faletra <palinuro@parrotsec.org>
Last-Update: 2019-06-19

--- firejail-0.9.58.2.orig/etc/libreoffice.profile
+++ firejail-0.9.58.2/etc/libreoffice.profile
@@ -18,7 +18,6 @@ noblacklist /etc/java
 noblacklist /usr/share/java
 
 include disable-common.inc
-include disable-devel.inc
 include disable-passwdmgr.inc
 include disable-programs.inc
 
@@ -34,22 +33,22 @@ netfilter
 nodvd
 nogroups
 # comment nonewprivs when using the ubuntu 18.04/debian 10 apparmor profile
-nonewprivs
+#nonewprivs
 noroot
 notv
 nou2f
 # comment the protocol line when using the ubuntu 18.04/debian 10 apparmor profile
-protocol unix,inet,inet6
+#protocol unix,inet,inet6
 # comment seccomp when using the ubuntu 18.04/debian 10 apparmor profile
-seccomp
+#seccomp
 shell none
 # comment tracelog when using the ubuntu 18.04/debian 10 apparmor profile
-tracelog
+#tracelog
 
 private-dev
 private-tmp
 
-noexec ${HOME}
+#noexec ${HOME}
 noexec /tmp
 
 join-or-start libreoffice
